src/main.rs

```diff
@@ -457,6 +457,10 @@ fn backtrace(
     let mut registers = Registers::new(lr, sp, core);
     println!("stack backtrace:");
     loop {
+        // FIXME: Add some sanity check for the PC value to avoid unwinding a corrupted stack.
+        // For example, PC should always have the LSb (thumb bit) set, and should be located within
+        // `.text` (it is possible to execute code from RAM dynamically, but we cannot unwind that).
```

Comment:
(it theory `.text` can be placed in RAM and it may or may not be modified on the fly.)